### DISCUSSION TITLE
Hactoberfest PR not credited after being merged

### DISCUSSION THREAD

**Discussion by SubratYeeshu** [abhpd/hacktoberfest2021#607] [General]
![image](https://user-images.githubusercontent.com/75444646/135719333-2a362643-6107-4b6c-85c9-5b59cea33fd1.png)

**Comment by priyesh-raj-singh** [comment]
Yes, It is showing excluded project even after being merged and accepted.

**Comment by abhpd** [comment]
Yes, they seem to have excluded most of the DSA based repositories. I have contacted them will update you guys as soon as I have any further information.

**Reply by anonymous372** [reply]
### Excluded Project
@abhpd *Help on this one*

**Reply by abhpd** [reply]
They haven't replied yet.

**Comment by Hics0000** [comment]
Showing Excluded Project 
@abhpd
